mountinfo/mountinfo_openbsd.go

```diff
@@ -1,22 +0,0 @@
-package mountinfo
-
-import "golang.org/x/sys/unix"
-
-func int8SliceToString(is []int8) string {
-	var bs []byte
-	for _, i := range is {
-		if i == 0 {
-			break
-		}
-		bs = append(bs, byte(i))
-	}
-	return string(bs)
-}
-
-func getMountinfo(entry *unix.Statfs_t) *Info {
-	var mountinfo Info
-	mountinfo.Mountpoint = int8SliceToString(entry.F_mntonname[:])
-	mountinfo.FSType = int8SliceToString(entry.F_fstypename[:])
-	mountinfo.Source = int8SliceToString(entry.F_mntfromname[:])
-	return &mountinfo
-}
```

Comment:
See my comment above; instead of removing this file, it's still needed to have separate implementations, but you can remove the `int8SliceToString()` function,

Perhaps we should also remove the intermediate `mountinfo` variable, and just return a literal (looks slightly cleaner);


```suggestion
func getMountinfo(entry *unix.Statfs_t) *Info {
	return &Info{
		Mountpoint: unix.ByteSliceToString(entry.F_mntonname[:]),
		FSType:     unix.ByteSliceToString(entry.F_fstypename[:]),
		Source:     unix.ByteSliceToString(entry.F_mntfromname[:]),
	}
}
```